protos/route.proto

```diff
@@ -354,16 +354,19 @@ message MirrorLog {
   string error_message = 2;
   string error_type = 3;
   double error_timestamp = 4;
+  int32 id = 5;
 }
 message ListMirrorLogsRequest {
   string flow_job_name = 1;
   string level = 2;
-  int32 page = 3;
-  int32 num_per_page = 4;
+  int32 num_per_page = 3;
```

Comment:
this will break other clients (internal usage), lets try to maintain proto compatibility.

Reply:
done, can wait for 0.19 to remove ugly support for page param